integration/ig/k8s/profile_tcprtt_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 The Inspektor Gadget authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"testing"
+
+	. "github.com/inspektor-gadget/inspektor-gadget/integration"
+	tcprttProfileTypes "github.com/inspektor-gadget/inspektor-gadget/pkg/gadgets/profile/tcprtt/types"
+	"github.com/inspektor-gadget/inspektor-gadget/pkg/histogram"
+)
+
+func TestProfileTCPRTT(t *testing.T) {
+	t.Parallel()
+
+	commands := []*Command{
+		{
+			Name: "ProfileTCPRTT",
+			Cmd:  "ig profile tcprtt -o json --timeout 10",
+			ExpectedOutputFn: func(output string) error {
+				expectedEntry := &tcprttProfileTypes.Report{
+					Histograms: []*tcprttProfileTypes.ExtendedHistogram{
+						tcprttProfileTypes.NewHistogram(
+							histogram.UnitMicroseconds,
+							nil,
+							tcprttProfileTypes.WildcardAddress,
+							tcprttProfileTypes.AddressTypeAll,
+							1,
+						),
+					},
+				}
+
+				normalize := func(r *tcprttProfileTypes.Report) {
+					if len(r.Histograms) != 1 {
+						return
+					}
+
+					r.Histograms[0].Intervals = nil
+
+					if r.Histograms[0].Average != 0 {
+						r.Histograms[0].Average = 1
+					}
+				}
+
+				return ExpectEntriesToMatch(output, normalize, expectedEntry)
+			},
+		},
+	}
+
```

Comment:
I suppose it works because there is some traffic being generated by another applications on the host, right?

IMO this should generate its own traffic to avoid relying on that.

Reply:
You're right. Done. Given that they are identical, I moved them to a shared folder.